src/EnergyPlus/ChillerExhaustAbsorption.cc

```diff
@@ -1791,9 +1791,23 @@ void ExhaustAbsorberSpecs::calcChiller(EnergyPlusData &state, Real64 &MyLoad)
             if (lCondWaterMassFlowRate > DataBranchAirLoopPlant::MassFlowTolerance) {
                 lCondSupplyTemp = lCondReturnTemp + lTowerLoad / (lCondWaterMassFlowRate * Cp_CD);
             } else {
-                ShowSevereError(state, "CalcExhaustAbsorberChillerModel: Condenser flow = 0, for Exhaust Absorber Chiller=" + this->Name);
-                ShowContinueErrorTimeStamp(state, "");
-                // ShowFatalError(state, "Program Terminates due to previous error condition.");
+                if (this->lCondWaterMassFlowRate_Index == 0) {
+                    ShowSevereError(state, "CalcExhaustAbsorberChillerModel: Condenser flow = 0, for Exhaust Absorber Chiller=" + this->Name);
+                    ShowContinueErrorTimeStamp(state, "");
+                    // ShowFatalError(state, "Program Terminates due to previous error condition.");
+                }
+                ShowRecurringSevereErrorAtEnd(state,
+                                              "CalcExhaustAbsorberChillerModel: Condenser flow = 0, for Exhaust Absorber Chiller=" + this->Name +
+                                                  ": Condenser flow rate = 0 sever error warning continues...", // Message automatically written to
+                                                                                                                // "error file" at end of simulation
```

Comment:
Also, I think it would be good if the unit test could test this `ShowRecurringSevereErrorAtEnd`, since that was the main point mentioned in [issue 8638](https://github.com/NREL/EnergyPlus/issues/8638).

Reply:
Agree--I will try adding a check on this.